### PR TITLE
Feature: v2 Tabs

### PR DESCRIPTION
**Pull request by blumendorf**

### CODE REVIEW

File: src/components/V2Tabs/V2Tabs.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Description
+
+*V2Tabs* render a tab-like navigation, with links that can point to the tab content.
+ 
+- `olt-V2Tabs` - The tab bar
+- `olt-V2Tabs-link` - The tab link
+
+The *V2Tabs* are centered on large screens and right aligned on small screens.
+The active tab via decorated via `is-active` state class name.
```
Comment (derlarsen) [inline src/components/V2Tabs/V2Tabs.md]: Typo here? `via decorated  via`

Comment (blumendorf) [reply src/components/V2Tabs/V2Tabs.md]: Fixed. Thanks.

File: src/components/V2Tabs/V2Tabs.scss [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    align-items: flex-start;
+    overflow-x: scroll;
+  }
+
+  &-link {
+    height: $tab-height;
+    width: auto;
+    border: 0;
+    background: $tab-inactive-bg-color; // ToDo: use variable (#0d122c)
+    border-radius: $tab-border-radius; // ToDo: use variable
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.scss]: `ToDos` can be removed?

File: src/components/V2Tabs/V2Tabs.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+To add a gradient to show that there are more elements available, use `has-gradient-left` and `has-gradient-right` state class name. The gradient overlay on either side is used to indicate that more content is available outside of the visible area.
+
+```newnavigationgradient.html
+<div class="olt-V2Tabs">
+  <a class="olt-V2Tabs-link is-active">Link One</a>
+  <a class="olt-V2Tabs-link">Link Two</a>
+  <a class="olt-V2Tabs-link">Link Three</a>
+  <a class="olt-V2Tabs-link">Link Four</a>
+  <a class="olt-V2Tabs-link">Link Five</a>
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.md]: Should have a few more elements to actually allow scrolling on the styleguide page.

File: src/components/V2Tabs/V2Tabs.scss (hunk 2 above)
Comment (blumendorf) [reply src/components/V2Tabs/V2Tabs.scss]: Yeah. missed those. Thanks.

File: src/components/V2Tabs/V2Tabs.md (hunk 3 above)
Comment (blumendorf) [reply src/components/V2Tabs/V2Tabs.md]: The design specifies this only:
Desktop Layout – Tabs are centered horizontally across the available space
Tablet Layout – Tabs are aligned left
Mobile Layout – Tabs are aligned left and scrollable

So scrolling should only happen in mobile mode. I did implemented it a bit more flexible though.

Comment (bra1n) [reply src/components/V2Tabs/V2Tabs.md]: I see, I tried it in responsive mode but had to make the window reaaaally small to get any scrolling, which would've been easier if there were more elements in the demo.

Comment (blumendorf) [reply src/components/V2Tabs/V2Tabs.md]: I added a modifier and another (longer) case.

File: src/components/V2Tabs/V2Tabs.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  <a class="olt-V2Tabs-link">Link Five</a>
+  <a class="olt-V2Tabs-link">Link Five</a>
+  <a class="olt-V2Tabs-link">Link Five</a>
+  <a class="olt-V2Tabs-link">Link Five</a>
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.md]: Is it intended to have 5 "Link Five" tabs?

File: src/components/V2Tabs/V2Tabs.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+var overflows = tabs.scrollWidth>tabs.clientWidth;
+if (overflows) tabs.classList.add("has-gradient-right");
+
+tabs.addEventListener('scroll', () => {
+  var hasScrolledLeft = tabs.scrollLeft > 0;
+  if (hasScrolledLeft) {
+    tabs.classList.add("has-gradient-left");
+  } else {
+    tabs.classList.remove("has-gradient-left");
+  }
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.md]: L67-72 can be simplified to: `tabs.classList.toggle("has-gradient-left", tabs.scrollLeft > 0);`

File: src/components/V2Tabs/V2Tabs.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+  var canScrollRight = tabs.scrollWidth-tabs.scrollLeft-tabs.clientWidth > 0;        
+  if (canScrollRight) {
+    tabs.classList.add("has-gradient-right");
+  } else {
+    tabs.classList.remove("has-gradient-right");
+  }
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.md]:
```suggestion
  tabs.classList.toggle("has-gradient-right", tabs.scrollWidth-tabs.scrollLeft-tabs.clientWidth > 0);
```

File: src/components/V2Tabs/V2Tabs.md (hunk 4 above)
Comment (blumendorf) [reply src/components/V2Tabs/V2Tabs.md]: Missed that. Thanks.

File: src/components/V2Tabs/V2Tabs.md (hunk 5 above)
Comment (blumendorf) [reply src/components/V2Tabs/V2Tabs.md]: Nice!

**Comment by bra1n** [comment]
Looks like you need to update dependencies.

**Pull request merged**